SamplesDashboard/SamplesDashboard/Policies/GitHubRetryPolicy.cs

```diff
@@ -17,12 +17,12 @@ private static IAsyncPolicy<HttpResponseMessage> GetPolicy()
         {
             return HttpPolicyExtensions
               .HandleTransientHttpError()
-              .OrResult(msg => msg.StatusCode == HttpStatusCode.Forbidden)
+              .OrResult(msg => msg.StatusCode is HttpStatusCode.Forbidden or HttpStatusCode.InternalServerError)
```

Comment:
https://github.com/App-vNext/Polly.Extensions.Http/blob/69fd292bc603cb3032e57b028522737255f03a49/src/Polly.Extensions.Http/HttpPolicyExtensions.cs#L12

Already caters for this.

Reply:
I've reverted the change